springfox-swagger-common/build.gradle

```diff
@@ -16,18 +16,25 @@ dependencies {
   compile project(':springfox-spring-web')
   compile libs.core
   compile libs.spring
+  compile group: 'org.springframework.boot', name: 'spring-boot-autoconfigure', version: springBoot
```

Comment:
This dependency is undesirable. Why is spring boot needed here?

Reply:
In `ApiResourceController` I introduced 2 controllers dealing with webMvc and webFlux respectively, to let the Spring be aware of which controller should be registered, the `@ConditionalOnClass` is needed, hence the `spring-boot-autoconfigure` here. Now that you've suggested another path, I will move the 2 different loaders to `springfox-spring-webmvc` and `springfox-spring-webflux` respectively and then register their default instance as the default loader to use into a mediator which is accessible in both `springfox-spring-webflux/webmvc` and `springfox-swagger-common`